inappmessaging/USERGUIDE.md

```diff
@@ -462,6 +462,7 @@ Documents targeting Product Managers:
 * SDKCF-5601: Fixed close button's content label accessibility warnings.
 * SDKCF-5900: Refactored code to remove most of the suppressions for code smells.
 * SDKCF-5948: Added tooltip campaigns feature.
+* SDKCF-6035: Added `closeTooltip()` API to manually close displayed tooltip by `UIElement` identifier.
```

Comment:
nit: maybe need to rename this since `UIElement` is used in iOS and not in Android.
suggestion: "viewId"

Reply:
I'm referring to the `UIElement` property set in campaign body which is used for iOS and Android both. Is it ok not to change it?

Reply:
Ah got it. But better to rename the API parameter name.

Reply:
updated parameter name